mne/time_frequency/multitaper.py

```diff
@@ -451,15 +451,14 @@ def _mt_spectra(x, dpss, sfreq, n_fft=None):
     return x_mt, freqs
 
 
-@verbose
-def multitaper_psd(x, sfreq=2 * np.pi, fmin=0, fmax=np.inf, bandwidth=None,
-                   adaptive=False, low_bias=True, n_jobs=1,
-                   normalization='length', verbose=None):
+def _psd_multitaper(x, sfreq, fmin=0, fmax=np.inf, bandwidth=None,
+                    adaptive=False, low_bias=True, normalization='length',
+                    n_jobs=1):
     """Compute power spectrum density (PSD) using a multi-taper method
 
     Parameters
     ----------
-    x : array, shape=(n_signals, n_times) or (n_times,)
+    x : array, shape=([n_epochs], n_signals, n_times) or (n_times,)
```

Comment:
Can't it just be `array, shape=(..., n_times)`? I think we really support anything where times is the last axis.

But for future reference, the couple of times I've hit this problem I've kept everything on the same line to be safe, and told the PEP8 checker to ignore it.